### ISSUE TITLE
Mailing List

### ISSUE THREAD

**Issue by cc7768** [numfocus/numfocus.org#134]
Discuss which mailing list or software we should use to facilitate communication across projects

**Comment by rgommers** [comment]
https://groups.google.com/forum/#!forum/numfocus would be good for everything that we'd want to discuss publicly. Or were you thinking a list specific for only sponsored projects?

**Comment by labarba** [comment]
What about a Discourse?
http://www.discourse.org

**Comment by cc7768** [comment]
This was a question that came up during the NumFOCUS summit.

I opened an issue as a place holder.

**Comment by rgommers** [comment]
@cc7768 do you have a suggestion? or maybe some thoughts on the requirements (like public or private, only for governance topics or also announcements or technical topics, etc.)?

**Comment by tracykteal** [comment]
I think the elements desired in the list were
- archival and able to be referenced (so something like discourse was better than email threads)
- for the NumFOCUS projects to communicate with each other

I think the ideas was that it would be publicly viewable.

I don't think we decided who could join, if it was limited to NumFOCUS projects or anyone could join.

Discourse seemed to be the consensus at the Summit meeting. rOpenSci has been using it and recommend it. Is this something we can have NumFOCUS host? Their hosting is expensive ($100/month), so can we get it setup on NumFOCUS servers?

**Comment by cc7768** [comment]
I think having a place for NumFOCUS projects to communicate amongst the projects would be useful. It could deal with both governance topics and technical issues -- They have a pretty decent tagging system, if I'm not mistaken, that could help us organize discussions about a variety of topics.

Do you recall whether rOpenSci was hosting it themselves or whether they were paying to have it hosted? [This](http://blog.discourse.org/2016/03/free-discourse-forum-hosting-for-community-friendly-github-projects/) blog post of theirs also seems relevant.

**Comment by rgommers** [comment]
> Their hosting is expensive ($100/month), so can we get it setup on NumFOCUS servers?

Since we have no servers, and more importantly no sysadmin, this doesn't sound like a good plan (apart from it being very expensive). We need a hosted solution.

**Comment by tracykteal** [comment]
Both rOpenSci and Data Carpentry have it set up on Digital Ocean. Some sys admin from our orgs to get it set up, but then just about $10 a month.

The free hosting option for open source seems ideal though!

**Comment by dpinte** [comment]
Based on experience from other projects (mostly numpy/scipy), we should push at all price to use a hosting solution. Hosting mailing list can be a real pain (spamming, virus, ddos attacks, etc.)

**Comment by rgommers** [comment]
Looked at this again, and had some private feedback today on my message saying "this numfocus mailing list is quite these days, look at Twitter to see what's going on". The current mailing list *is* too quiet. So is the project mailing list. Both of those are decent mediums for communication between projects, so I don't want to set up yet another forum that's then again going to be too quiet. First start communicating more, then when there's a need (i.e. mailing list gets too hard to follow) we can consider other alternatives.

**Comment by cc7768** [comment]
That seems fair. Let's go ahead and close it then.

**Comment by rgommers** [comment]
Thanks @cc7768, closing then. We can always reopen later.
